File: user_spring/src/main/java/com/kjj/user/dto/order/OrderDto.java
package com.kjj.user.dto.order;

import com.kjj.user.entity.Order;
import com.kjj.user.util.DateUtil;
import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public class OrderDto {

    private Long id;

    private String menu;
    private int cost;
    private String orderDate;
    private boolean recognize;
    private boolean expired;
    private boolean payment;

    public static OrderDto from(Order order) {
        return new OrderDto(
                order.getId(),
                order.getMenu(),
                order.getCost(),
                DateUtil.makeLocaldateToFormatterString(order.getOrderDate()),
                order.isRecognize(),
                order.isExpired(),
                order.isPayment()
        );
    }
}

File: user/src/main/kotlin/com/kjj/user/dto/order/OrderDto.kt
package com.kjj.user.dto.order

import com.kjj.user.entity.Order
import com.kjj.user.util.DateUtil

data class OrderDto(
    val id: Long?,
    val menu: String,
    val cost: Int,
    val orderDate: String,
    val recognize: Boolean,
    val expired: Boolean,
    val payment: Boolean
) {
    companion object {
        fun from(order: Order): OrderDto {
            return OrderDto(
                order.id,
                order.menu,
                order.cost,
                DateUtil.makeLocalDateToFormatterString(order.orderDate),
                order.recognize,
                order.expired,
                order.payment
            )
        }
    }
}